iidm/iidm-api/src/main/java/com/powsybl/iidm/network/IdBasedBusRef.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.network;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Objects;
+import java.util.Optional;
+
+/**
+ * @author Yichen TANG <yichen.tang at rte-france.com>
+ */
+public class IdBasedBusRef extends AbstractVoltageLevelBasedBusRef {
```

Comment:
So if I understand well: here the `BusRef` is based on the `id` of a bus from the bus view ? And I can use it to define a sensitivity factor? But, as we will perform a contingency analysis too in the sensitivity analysis, the buses from the bus view are not stable. In the OLF, the a `AC` sensitivity analysis is based on the bus/breaker view. So, it is really complex and dangerous to use this implementation of `BusRef` to model a sensitivity factor.

Reply:
I think the use of that implementation is valid for bus breaker voltage levels (which use the "stable" `ConfiguredBus`). 
we should throw for node breaker voltage levels.

Reply:
I think I am understanding that:
```    
@Override
    public Optional<Bus> resolve(Network network) {
        return safeGetVoltageLevel(network)
                .map(VoltageLevel::getBusView)
                .map(busView -> busView.getBus(busId));
    }
```